src/MongoDB/CursorId.c

```diff
@@ -58,14 +58,14 @@ static bool php_phongo_cursorid_init_from_hash(php_phongo_cursorid_t* intern, Ha
 	return false;
 } /* }}} */
 
-static HashTable* php_phongo_cursorid_get_properties_hash(phongo_compat_object_handler_type* object, bool is_debug) /* {{{ */
+static HashTable* php_phongo_cursorid_get_properties_hash(phongo_compat_object_handler_type* object, bool is_temp, bool is_serialize) /* {{{ */
```

Comment:
The existing implementation of `Serializable::serialize()` always used `ADD_ASSOC_INT64_AS_STRING`. That didn't seem necessary if the value falls within the range of a 32-bit signed integer. For example, `WriteConcern::serialize()` encodes `wtimeout` as either an integer or string based on its range.

However, `php_phongo_cursorid_init_from_hash` currently only expects a string, so I'll either need to change that to also accept an integer or remove the conditional encoding for `is_serialize` below.

Reply:
Opted to remove conditional encoding for `is_serialize` since that always maintains existing behavior before this PR. See my comment below.